### DISCUSSION TITLE
how to configure and enable imap port 143 on postal mail server

### DISCUSSION THREAD

**Discussion by selfiestick03** [postalserver/postal#1913] [Help with using Postal] [answered]
I research on web but not got any method to configure and enable imap port 143 on my postal mail server and what should i edit in postal.yml file to enable 143 port work on postal

**Comment by willpower232** [comment, marked as answer]
Postal is not a traditional email server with mailboxes so does not have any IMAP functionality. Postals main purpose is to allow other software and devices to send emails.

**Reply by JustinBack** [reply]
Interesting, how does receiving emails work without any IMAP server?

**Reply by willpower232** [reply]
if the originating system needs to receive messages, it can be done with routing to SMTP or API endpoints but Postal mostly just sends emails
